src/Select/SelectInput.js

```diff
@@ -343,6 +349,14 @@ class SelectInput extends React.Component<AllProps, State> {
             ...MenuProps.MenuListProps,
```

Comment:
I think that this logic is broken.

Reply:
I didn't even touch this line. It seems reasonable to me. :+1:

Reply:
I have the feeling that if a user provides a `MenuListProps` property the merge won't happen correctly.

Reply:
I think it will merge correctly, but I'll verify the logic.

Reply:
My bad.